### PR TITLE
Longest Palindromic Substring

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Problem: To find the Longest Palindrome Substring from given string.

Example:
	Input:
	
	Enter String: Boon
	
	Output:
	Longest palindrome substring is:
	oo
	Length is: 2

Time Complexity: O(n*2)
Space Complexity: O(1)

### Checklist

- [Y ] I've read the contribution guidelines.
- [ Y] I've checked the issue list before deciding what to submit.
- [ Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#632 
ProgramOutput

### CODE REVIEW

**Comment by narayan2111** [comment]
Kindly Review this PR @iamrajiv @plazzy99 @SKAUL05 @Abhijit2505 @siddharth25pandey

File: Python/cp/Longest_Palindrome_Substring.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+'''
+To find the longest Palindrome Substring in given string
+
+Sample I/O:
+	Input:
+	
+	Enter String: Boon
+	
+	Output:
+	Longest palindrome substring is:
```
Comment (vatsalkesarwani12) [inline Python/cp/Longest_Palindrome_Substring.py]: move this comment to the end

**Review by vatsalkesarwani12 — APPROVED** [review]
LGTM 👍

File: Python/cp/Longest_Palindrome_Substring.py (hunk 1 above)
Comment (narayan2111) [reply Python/cp/Longest_Palindrome_Substring.py]: Done Sir.

**Review by vatsalkesarwani12 — APPROVED** [review]

File: Python/README.md [hunk 2]
```diff
@@ -1,4 +1,406 @@
 
+
+<p align="center">
```
Comment (atarax665) [inline Python/README.md]: You have made so many changes to README.md. Revert all the changes and just add hyperlink of your program.

File: Python/cp/Longest_Palindrome_Substring.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+def palindrome(string):
```
Comment (atarax665) [inline Python/cp/Longest_Palindrome_Substring.py:10]: Add description of the problem at the top in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

File: Python/cp/Longest_Palindrome_Substring.py (hunk 3 above)
Comment (narayan2111) [reply Python/cp/Longest_Palindrome_Substring.py:1]: But Sir, Vatsal Sir told me to bring it down @atarax665

File: Python/README.md (hunk 2 above)
Comment (narayan2111) [reply Python/README.md]: Done sir

**Comment by narayan2111** [comment]
Kindly look to this PR @atarax665

**Review by atarax665 — CHANGES_REQUESTED** [review]
Edit the README file to link your contribution, else everything looks good to merge.

**Comment by narayan2111** [comment]
> Edit the README file to link your contribution, else everything looks good to merge.

Done sir @atarax665

**Review by atarax665 — APPROVED** [review]

**Pull request merged**